src/runtime/pythonengine.cs

```diff
@@ -330,11 +328,26 @@ public static void Shutdown()
 
             PyScopeManager.Global.Clear();
             ExecuteShutdownHandlers();
+            // Remember to shut down the runtime.
+            Runtime.Shutdown(mode);
```

Comment:
`Runtime.Shutdown` called in `ExecuteShutdownHandlers`

Reply:
I removed it as a ShutdownHandler and put it here, so that if the runtime was Initialized on Reload, but we want to shutdown with a Normal mode because the application is quitting, then we can do so.

For example, in the Unity Editor the `ShutdownMode` would always be `Reload`. Eventually, when the application quits, we want to completely shutdown the runtime.